crates/turbo-tasks-fetch/src/lib.rs

```diff
@@ -153,29 +153,31 @@ impl Issue for FetchIssue {
     }
 
     #[turbo_tasks::function]
-    async fn description(&self) -> Result<Vc<StyledString>> {
+    async fn description(&self) -> Result<Vc<OptionStyledString>> {
```

Comment:
Wouldn't it be easier to just add `StyledString::Nothing`?

Reply:
Yeah, I considered this but thought it might be more expected/idiomatic to use a native `Option`, though that adds an additional Vc.